pages/common/live-server.md

```diff
@@ -0,0 +1,19 @@
+# live-server
+
+> A simple development http server with live reload capability.
+
+- Serve an index.html file and reload on changes:
+
+`live-server`
+
+- Specify a port from which to serve a file:
```

Comment:
Same question here.

Reply:
I'm unclear on what the issue with this line is. The default port from which live-server is run is http://localhost:8080, you can change that to whatever port you'd like.

Reply:
Ah its 8080. In that case, perhaps you can mention it in brackets beside port like `Specify a port(default is 8080) from which to serve a file`